ratis-server-api/src/main/java/org/apache/ratis/server/DataStreamServerRpc.java

```diff
@@ -20,18 +20,12 @@
 import org.apache.ratis.protocol.RaftPeer;
 
 import java.io.Closeable;
-import java.net.InetSocketAddress;
 
 /**
  * A server interface handling incoming streams
  * Relays those streams to other servers after persisting
  */
-public interface DataStreamServerRpc extends RaftPeer.Add, Closeable {
-  /**
-   * start server
-   */
+public interface DataStreamServerRpc extends ServerRpc, RaftPeer.Add, Closeable {
+  @Override
   void start();
```

Comment:
Do we need to keep this function as ServerRpc already has one?

Reply:
You are right that we can remove it, although this signature in this one is different -- it does not `throws IOException`.